packages/swap/src/handlers/openSwapDepositChannel.ts

```diff
@@ -53,6 +53,12 @@ export default async function openSwapDepositChannel(
     throw ServiceError.badRequest('too many channels');
   }
 
+  // DEPRECATED(1.5): use ccmParams instead of ccmMetadata
+  const deprecatedCcmMetadata = (input as { ccmMetadata?: CcmParams }).ccmMetadata;
+  if (!input.ccmParams && deprecatedCcmMetadata) {
+    input.ccmParams = deprecatedCcmMetadata; // eslint-disable-line no-param-reassign
+  }
```

Comment:
if we keep both in the schema, this stuff is simplified
```suggestion
  input.ccmParams ??= input.ccmMetadata;
```